pkg/resource/deploy/deployment_executor.go

```diff
@@ -477,7 +518,6 @@ func (ex *deploymentExecutor) refresh(callerCtx context.Context, opts Options, p
 	}
 
 	// Make sure if there were any targets specified, that they all refer to existing resources.
-	targetMapOpt := createTargetMap(opts.RefreshTargets)
```

Comment:
I could revert this, just moved it down 50 lines to where it was actually used while I was looking at this code a lot.